ts/packages/lib/src/integrations/store.ts

```diff
@@ -43,6 +45,8 @@ export const STORE_WEBSITE: Record<StoreType, string> = {
   configu: 'https://configu.com/',
   'json-file': 'https://www.json.org/json-en.html',
   'hashicorp-vault': 'https://www.vaultproject.io/',
+  'aws-parameter-store':
```

Comment:
change to the main product page as in secret-manager please

Reply:
There is no main product page as it's a feature of AWS Systems Manager. There is [this section](https://aws.amazon.com/systems-manager/features/#Parameter_Store) on the product page but it doesn't really link to anything useful in the context of the parameter store. What would you prefer?